include/guess/string_util.h

```diff
@@ -56,19 +53,26 @@ inline void normalize(std::string& s) {
   replace_all(s, "(", " ");
   replace_all(s, ")", " ");
 
-  for (int i = 0; i < s.length(); ++i) {
-    char c = s[i];
-    bool is_number = c >= '0' && c <= '9';
-    bool is_lower_case_char = c >= 'a' && c <= 'z';
-    bool is_upper_case_char = c >= 'A' && c <= 'Z';
-    if (!is_number && !is_lower_case_char && !is_upper_case_char) {
-      s[i] = ' ';
-    }
+  auto options = static_cast<utf8proc_option_t>(
+      utf8proc_option_t::UTF8PROC_DECOMPOSE |
+      utf8proc_option_t::UTF8PROC_STRIPMARK |
+      utf8proc_option_t::UTF8PROC_CASEFOLD);
+
+  utf8proc_uint8_t *utf8_out = nullptr;
```

Comment:
```suggestion
  utf8proc_uint8_t* utf8_out = nullptr;
```

Are you using `clang-format` with the `.clang-format` file?
This looks like it's formatted with a different style.